docs/Simorgh-Release-Info.md

```diff
@@ -4,51 +4,51 @@ Remember to commit and push the changes to both launchDates.js and Simorgh-Relea
-| uzbek | [home](http://localhost:7080/uzbek)<br/>[articles](http://localhost:7080/uzbek/articles/cxj3rjxm6r0o)<br/>[liveRadio](http://localhost:7080/uzbek/bbc_uzbek_radio/liveradio)<br/>[MAP](http://localhost:7080/uzbek/world-23053613)<br/>[PGL](http://localhost:7080/uzbek/central-asia-46716844)<br/>[mostRead](http://localhost:7080/uzbek/popular/read)<br/>[onDemandRadio](http://localhost:7080/uzbek/bbc_uzbek_radio/w172x9f9qjcq3lm) | [home](https://www.test.bbc.com/uzbek)<br/>[liveRadio](https://www.test.bbc.com/uzbek/bbc_uzbek_radio/liveradio)<br/>[MAP](https://www.test.bbc.com/uzbek/world-23053613)<br/>[PGL](https://www.test.bbc.com/uzbek/sport-23098743)<br/>[mostRead](https://www.test.bbc.com/uzbek/popular/read) | [home](https://www.stage.bbc.com/uzbek)<br/>[liveRadio](https://www.stage.bbc.com/uzbek/bbc_uzbek_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/uzbek/world-23053613)<br/>[PGL](https://www.stage.bbc.com/uzbek/sport-23098743)<br/>[mostRead](https://www.stage.bbc.com/uzbek/popular/read) | [home](https://www.bbc.com/uzbek)<br/>[liveRadio](https://www.bbc.com/uzbek/bbc_uzbek_radio/liveradio)<br/>[MAP](https://www.bbc.com/uzbek/media-50461363)<br/>[PGL](https://www.bbc.com/uzbek/central-asia-46716844)<br/>[mostRead](https://www.bbc.com/uzbek/popular/read) | __Home__: 13th Jan 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 4th Feb 2020<br/>__MAP__: 26th March 2020<br/>__PGL__: 12th March 2020<br/>__Most Read__: 28th May 2020 |
+| ukrainianRu | [IDX](http://localhost:7080/ukrainian/ukraine_in_russian) | [IDX](https://www.test.bbc.com/ukrainian/ukraine_in_russian) | [IDX](https://www.stage.bbc.com/ukrainian/ukraine_in_russian) | [IDX](https://www.bbc.com/ukrainian/ukraine_in_russian) |  |
```

Comment:
This puts the Ukrainian index in its own row as the service name is listed differently in our config..

Reply:
Oh I didn't refresh the page, you've already clocked it 😄